hera_cal/vis_clean.py

```diff
@@ -213,6 +441,77 @@ def apply_calibration(self, input_cal, unapply=False):
         apply_cal.calibrate_in_place(self.data, cal_gains, self.flags, cal_flags,
                                      gain_convention=gain_convention)
 
+    def apply_flags(self, external_flags=None, overwrite_data_flags=False,
```

Comment:
I'm still not enthused about how this is done, in the sense that it's not as consistent with `apply_calibration` as I would like it. I understand that we want to use convenience methods that operate on UVData subclasses. I'll try to see if I can make this any more elegant--I'll try to get back to you shortly on this. thanks

Reply:
Thanks for the feedback. I've followed your recommended template and added an additional arg to support `yaml` files.